CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+Unknown
```

Comment:
I know this won't be in 7.45, so putting this here in order to create a merge conflict when this gets merged to `main`. At that point, I'll add this release note under the correct release heading.